### PR TITLE
Add new license Anti-Capitalist Software License #2362

### LINKED ISSUE #2362: Incorrect License detected for Anti-Capitalist Software License
## Incorrect License name Waterfall Feed Parser License has been detected for Anti-Capitalist Software License

- license scan 
- new feature

### PR DESCRIPTION
**Pull request by sritasngh**
Signed-off-by: itssingh saritasingh.0425@gmail.com
Fixes #2362

 Added new licenses ANTI-CAPITALIST SOFTWARE LICENSE v-1.0, v-1.1, v-1.2, v-1.3 and v-1.4
### These added licenses are giving correct scan result. 

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: src/licensedcode/data/licenses/anti-capitalist-1.0.LICENSE [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Copyright © [year] [copyright holders]
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.0.LICENSE]: Please do not include a "dummy" copyright template in the license text 
```suggestion
```

File: src/licensedcode/data/licenses/anti-capitalist-1.0.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+key: anti-capitalist-1.0
+short_name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.0)
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.0)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+text_urls: 
+    - https://anticapitalist.software/past_versions/v1-0.html
+ignorable_urls: 
+    - https://anticapitalist.software/past_versions/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.0.yml]: Can you add a trailing line return?

File: src/licensedcode/data/licenses/anti-capitalist-1.1.LICENSE [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Copyright © [year] [copyright holders]
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.1.LICENSE]: Same above: no copyright template there
```suggestion
```

File: src/licensedcode/data/licenses/anti-capitalist-1.2.LICENSE [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Copyright © [year] [copyright holders]
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.2.LICENSE]: Same as above.
```suggestion
```

File: src/licensedcode/data/licenses/anti-capitalist-1.3.LICENSE [hunk 5]
```diff
@@ -0,0 +1,20 @@
+ANTI-CAPITALIST SOFTWARE LICENSE (v 1.3)
+
+Copyright © [year] [copyright holders]
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.3.LICENSE]:
```suggestion
```

File: src/licensedcode/data/licenses/anti-capitalist-1.4.LICENSE [hunk 6]
```diff
@@ -0,0 +1,21 @@
+ANTI-CAPITALIST SOFTWARE LICENSE (v 1.4)
+
+Copyright © [year] [copyright holders]
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.4.LICENSE]:
```suggestion
```

File: src/licensedcode/data/licenses/anti-capitalist-1.3.yml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+key: anti-capitalist-1.3
+short_name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.3)
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.3)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+text_urls: 
+    - https://anticapitalist.software/past_versions/v1-3.html
+ignorable_urls: 
+    - https://anticapitalist.software/past_versions/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.3.yml]: Add trailing LF

File: src/licensedcode/data/licenses/anti-capitalist-1.3.LICENSE [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+2. The User is one of the following:
+    a. An individual person, working for themselves
+    b. An non-profit organization
+    c. An organization that seeks shared profit for all of its members and allows non-members to set the cost of their  labor
+
+3. If the User is an organization with owners, then all owners are workers and all workers are owners with equal equity and/or equal vote.
+
+4. If the User is an organization, then the User is not law enforcement or military, or working under either.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT EXPRESS OR IMPLIED WARRANTY OF ANY KIND, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.3.LICENSE]: Add trailing LF

File: src/licensedcode/data/licenses/anti-capitalist-1.2.yml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+key: anti-capitalist-1.2
+short_name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.2)
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.2)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+text_urls: 
+    - https://anticapitalist.software/past_versions/v1-2.html
+ignorable_urls: 
+    - https://anticapitalist.software/past_versions/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.2.yml]: Add trailing LF

File: src/licensedcode/data/licenses/anti-capitalist-1.2.LICENSE [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+1. The above copyright notice and this permission notice shall be included in all copies of the Software.
+2. The User is one of the following:
+    a. An individual person
+    b. An organization of people that does not seek profit
+    c. An organization of people that seeks shared profit for all its members and does not exploit the labor of non-members
+3. If the User is an organization with owners, then all owners are workers and all workers are owners with equal share.
+4. If the User is an organization, then the User is not law enforcement or military, or working under either.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT EXPRESS OR IMPLIED WARRANTY OF ANY KIND, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.2.LICENSE]: Add trailing LF

File: src/licensedcode/data/licenses/anti-capitalist-1.1.yml [hunk 11]
```diff
@@ -0,0 +1,10 @@
+key: anti-capitalist-1.1
+short_name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.1)
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.1)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+text_urls: 
+    - https://anticapitalist.software/past_versions/v1-1.html
+ignorable_urls: 
+    - https://anticapitalist.software/past_versions/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.1.yml]: Add trailing LF

File: src/licensedcode/data/licenses/anti-capitalist-1.1.LICENSE [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+
+1. The above copyright notice and this permission notice shall be included in all copies of the Software.
+2. The User is one of the following:
+    a. An individual person
+    b. An organization of people that does not seek profit
+    c. An organization of people that seeks shared profit for all its members and does not exploit the labor of          non-members
+3. If the User is an organization with owners, then all owners are workers and all workers are owners with equal share.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT EXPRESS OR IMPLIED WARRANTY OF ANY KIND, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.1.LICENSE]: Add trailing LF

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks! I have a few nitpickings for your consideration before we can merge.

**Review by pombredanne — CHANGES_REQUESTED** [review]
@itssingh  Thank you! Almost there too here!

See the "ignorables"-related test failures in https://dev.azure.com/nexB/scancode-toolkit/_build/results?buildId=3262&view=logs&j=3c16b29b-eb82-5c7a-26ca-153f5e2ef8d4&t=525ae439-c19a-5d45-abf1-df81b3df2f93&l=43551 for your review.

**Comment by sritasngh** [comment]
> See the "ignorables"-related test failures

should I put `url` in `notes:` ?

File: src/licensedcode/data/licenses/anti-capitalist-1.4.yml [hunk 13]
```diff
@@ -0,0 +1,9 @@
+key: anti-capitalist-1.4
+short_name: ACSL v. 1.4
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.4)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+ignorable_urls:
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.4.yml]:
```suggestion
```

File: src/licensedcode/data/licenses/anti-capitalist-1.4.yml [hunk 14]
```diff
@@ -0,0 +1,9 @@
+key: anti-capitalist-1.4
+short_name: ACSL v. 1.4
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.4)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+ignorable_urls: 
+    - https://anticapitalist.software/past_versions/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.4.yml]:
```suggestion
```

File: src/licensedcode/data/rules/anti_capitalist1.yml [hunk 15]
```diff
@@ -0,0 +1,5 @@
+license_expression: anti-capitalist-1.4
+is_license_text: yes
+ignorable_urls: 
+    - https://anticapitalist.software/past_versions/
```
Comment (pombredanne) [inline src/licensedcode/data/rules/anti_capitalist1.yml]:
```suggestion
```

File: src/licensedcode/data/rules/anti_capitalist1.yml [hunk 16]
```diff
@@ -0,0 +1,5 @@
+license_expression: anti-capitalist-1.4
+is_license_text: yes
+ignorable_urls:
```
Comment (pombredanne) [inline src/licensedcode/data/rules/anti_capitalist1.yml]:
```suggestion
```

**Review by pombredanne — COMMENTED** [review]
> should I put url in notes: 

no, you just need to add `ignorable_*` fields for URLs and emails

File: src/licensedcode/data/licenses/anti-capitalist-1.4.yml [hunk 17]
```diff
@@ -0,0 +1,8 @@
+key: anti-capitalist-1.4
+short_name: ACSL v. 1.4
+name: ANTI-CAPITALIST SOFTWARE LICENSE (v 1.4)
+category: Free Restricted
+owner: Everest Pipkin and Ramsey Nasser
+homepage_url: https://anticapitalist.software/
+
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.4.yml]:
```suggestion
spdx_license_key: LicenseRef-scancode-anti-capitalist-1.4
```

File: src/licensedcode/data/licenses/anti-capitalist-1.4.yml [hunk 18]
```diff
@@ -0,0 +1,8 @@
...
+
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anti-capitalist-1.4.yml]:
```suggestion
```

File: src/licensedcode/data/rules/anti_capitalist1.yml [hunk 19]
```diff
@@ -0,0 +1,3 @@
+license_expression: anti-capitalist-1.4
+is_license_text: yes
+
```
Comment (pombredanne) [inline src/licensedcode/data/rules/anti_capitalist1.yml]:
```suggestion
```

File: src/licensedcode/data/rules/anti_capitalist2.yml [hunk 20]
```diff
@@ -0,0 +1,3 @@
+license_expression: anti-capitalist-1.4
+is_license_text: yes
+
```
Comment (pombredanne) [inline src/licensedcode/data/rules/anti_capitalist2.yml]:
```suggestion
```

File: src/licensedcode/data/rules/anti_capitalist3.yml [hunk 21]
```diff
@@ -0,0 +1,3 @@
+license_expression: anti-capitalist-1.4
+is_license_text: yes
+
```
Comment (pombredanne) [inline src/licensedcode/data/rules/anti_capitalist3.yml]:
```suggestion
```

File: src/licensedcode/data/rules/anti_capitalist4.yml [hunk 22]
```diff
@@ -0,0 +1,3 @@
+license_expression: anti-capitalist-1.4
+is_license_text: yes
+
```
Comment (pombredanne) [inline src/licensedcode/data/rules/anti_capitalist4.yml]:
```suggestion
```

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thank you! Please consider the few cleanups I suggested. Also feel free to squash all your commits in a single one.

**Comment by sritasngh** [comment]
> consider the few cleanups I suggested

I left these https://github.com/nexB/scancode-toolkit/pull/2364#discussion_r644868769 as trailing line return        https://github.com/nexB/scancode-toolkit/pull/2364#discussion_r583924347 as you previously suggested. Removed now :)

**Comment by sritasngh** [comment]
@pombredanne Should I add notes for all the rules?

**Comment by pombredanne** [comment]
FYI I merged the latest develop and fixed the tests, then pushed back to your branch

**Review by pombredanne — APPROVED** [review]
LGTM. Merging now at last. Thanks!

**Pull request merged**